release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -2,27 +2,14 @@
 layout: ballerina-blank-page
 title: Release Note
 ---
-### Overview of Ballerina Swan Lake Alpha3
+<em>This is the third alpha in a series of planned alphas and betas leading up to the Ballerina Swan Lake GA release.</em> 
```

Comment:
```suggestion
<em>This is the third alpha in a series of planned alphas and betas leading up to the Ballerina Swan Lake GA release.</em> 
```
I don't think we need to emphasize this sentence in italic using the `em` tag. The current wording also sounds fine as it doesn't emphasize too much on the milestones and gives the idea and meaning straight away.

@MaryamZi - WDYT?

Reply:
+1 from me to emphasize the version a bit since these aren't yet the final release. But shouldn't we say

```suggestion
<em>This is the third alpha release in a series of planned alpha and beta releases leading up to the Ballerina Swan Lake GA release.</em> 
```

Reply:
It's just a style choice :). I was going for a casual voice. Also note the repetition of the word 'release', which was what I was trying to avoid. But since the rest of the release note is pretty formal, we can skip this and revisit the template for the next release for better user engagement with a semi-formal tone.

Reply:
Changed as follows: "This Ballerina Swan Lake <VERSION> introduces new language features planned for the Swan Lake GA release. Moreover, this release includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the Swan Lake <PREVIOUS_VERSION> release."